tests/standalone_plugins/test_file_extensions.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+import sys
+import tempfile
+import unittest
+from argparse import Namespace
+from io import StringIO
+from pathlib import Path
+from unittest.mock import patch
+
+from troubadix.standalone_plugins.file_extensions import (
+    check_extensions,
+    main,
+    parse_args,
+)
+
+
+class TestFileExtensions(unittest.TestCase):
+    def test_ok(self):
+        with tempfile.TemporaryDirectory() as tmpdir:
+            tempfile.mkstemp(dir=tmpdir, suffix=".nasl")
+            tempfile.mkstemp(dir=tmpdir, suffix=".inc")
+            child_dir = tempfile.mkdtemp(dir=tmpdir)
+            tempfile.mkstemp(dir=child_dir, suffix=".nasl")
```

Comment:
```suggestion
            tempfile.mkstemp(dir=child_dir, suffix=".nasl")
            tempfile.mkstemp(dir=child_dir, suffix=".inc")
```

to cover both cases or am i wrong?